code/__DEFINES/mobs.dm

```diff
@@ -383,8 +383,6 @@
 #define SHOCK_SUPPRESS_MESSAGE (1 << 4)
 /// No skeleton animation if a human was shocked
 #define SHOCK_NO_HUMAN_ANIM (1 << 5)
-/// Ignores TRAIT_STUNIMMUNE
-#define SHOCK_IGNORE_IMMUNITY (1 << 6)
```

Comment:
Why remove it instead of making it not-unused as intended?

Reply:
because actually fixing it is out of scope. this is just to make it not misleading

Reply:
removing it is equally out of scope as fixing it